jvm-libs/generic/json-rpc/src/test/kotlin/net/consensys/linea/jsonrpc/client/VertxHttpJsonRpcClientTest.kt

```diff
@@ -352,7 +356,7 @@ class VertxHttpJsonRpcClientTest {
 
     val timer =
       meterRegistry.timer(
-        "jsonrpc.request",
+        "linea.jsonrpc.request",
```

Comment:
What is the value of `linea` prefix here? This JSON rpc client is agnostic an could be used outside linea apps. Furthermore, adding the prefix now is a breaking change, so all the dashboards using this will break. 

We need to carefully access these changes.

IIRC, The metrics are already under a component/cluster which shall provide enough scope, I don't see the need for `linea` here.